src/components/progress-tracker/progress-tracker.pw.tsx

```diff
@@ -0,0 +1,450 @@
+            showDefaultLabels
+          />
+        );
+
+        const progressTrackerCustomValuePrepositionElement = progressTrackerCustomValuePreposition(
+          page
+        );
+        await expect(progressTrackerCustomValuePrepositionElement).toHaveText(
+          customValuePreposition
+        );
+      });
+    }
+  );
+
+  ([
+    ["top", 0, 2],
+    ["bottom", 1, 3],
+  ] as [ProgressTrackerProps["labelsPosition"], number, number][]).forEach(
+    ([labelsPosition, minValIndex, maxValIndex]) => {
+      test(`render component with labelsPosition is set to ${labelsPosition}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(
+          <ProgressTrackerComponent labelsPosition={labelsPosition} />
+        );
+
+        const progressTrackerMinValElementIndex = progressTrackerMinVal(
+          page,
+          minValIndex
+        );
+        await expect(progressTrackerMinValElementIndex).toHaveText("50%");
+
+        const progressTrackerMaxValElementIndex = progressTrackerMaxVal(
+          page,
+          maxValIndex
+        );
+        await expect(progressTrackerMaxValElementIndex).toHaveText("100%");
+      });
+    }
+  );
+
+  ([CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS] as const).forEach(
+    (description) => {
+      test(`render component with description prop set to ${description}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(
+          <ProgressTrackerComponent
+            showDefaultLabels
+            description={description}
+          />
+        );
+
+        const progressTrackerDescriptionElement = progressTrackerDescription(
+          page
+        );
+        await expect(progressTrackerDescriptionElement).toHaveText(description);
+      });
+    }
+  );
+
+  test("has the expected border radius styling", async ({ mount, page }) => {
+    await mount(<ProgressTrackerComponent progress={35} />);
+
+    const progressTrackerLineElementParent = progressTrackerLine(page).locator(
+      ".."
+    );
+    await expect(progressTrackerLineElementParent).toHaveCSS(
+      "border-radius",
+      "32px"
+    );
+    const progressTrackerLineElement = progressTrackerLine(page);
+    await expect(progressTrackerLineElement).toHaveCSS("border-radius", "32px");
+  });
+});
+
+test.describe("Accessibility tests", () => {
+  ([
+    "aria-label",
+    "aria-describedby",
+    "aria-valuenow",
+    "aria-valuemin",
+    "aria-valuemax",
+    "aria-valuetext",
+  ] as const).forEach((propName) => {
+    test.describe(`when ${propName} prop is passed`, () => {
+      test(`check the accessibility when the ${propName} is set to cypress-standard`, async ({
+        mount,
+        page,
+      }) => {
+        const isNowMinOrMax = checkPropName(propName);
+        const props = getProps(propName, isNowMinOrMax);
+        await mount(<ProgressTrackerComponent {...props} />);
+
+        await checkAccessibility(page);
+      });
+    });
+  });
```

Comment:
**suggestion**: As these ARIA attributes related to progress trackers are more useful when used collectively in an example use case, what do you think about us copying these storybook stories into `component-test.pw.tsx`:

- ["accessibility example" story](https://carbon.sbc-ui.com/?path=/story/progress-tracker--accessibility-example)
- ["accessibility example 2" story](https://carbon.sbc-ui.com/?path=/story/progress-tracker--accessibility-example-two)

then creating two `checkAccessibility` tests for each?